### PR TITLE
OCPBUGS-18432: Updated TuneD bootloader supportability note

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.10+

Issue:
https://issues.redhat.com/browse/OCPBUGS-18432

Link to docs preview (VPN required):
[Supported TuneD daemon plugins](https://file.rdu.redhat.com/antaylor/OCPBUGS-18432/scalability_and_performance/using-node-tuning-operator.html#supported-tuned-daemon-plug-ins_node-tuning-operator)

QE review:
- [x] QE has approved this change.

Additional information:
Corrected a note and adjusted it from a WARNING to a NOTE block in two places.

### CODE REVIEW

**Comment by jmencak** [comment]
Thank you for the PR, Andrew.  Let's be more generic.  We support bootcmdline on 8.x and 9.x.  So perhaps omit 8.x and just say we support them on RHCOS worker nodes?

**Comment by jmencak** [comment]
/lgtm 
Thank you.

**Review by liqcui — COMMENTED** [review]
/lgtm

File: modules/configuring-huge-pages.adoc [hunk 1]
```diff
@@ -92,9 +92,9 @@ $ oc get node <node_using_hugepages> -o jsonpath="{.status.allocatable.hugepages
 ----
 
 ifndef::openshift-origin[]
-[WARNING]
+[NOTE]
 ====
-The TuneD bootloader plugin is currently supported on {op-system-first} 8.x worker nodes. For {op-system-base-full} 7.x worker nodes, the TuneD bootloader plugin is currently not supported.
+The TuneD bootloader plugin is only supported on {op-system-first} worker nodes.
```
Comment (tmalove) [inline modules/configuring-huge-pages.adoc:97]:
```suggestion
The TuneD boot loader plugin only supports {op-system-first} worker nodes.
```
See SSG: https://redhat-documentation.github.io/supplementary-style-guide/#boot-loader
The other suggestion is for active voice: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#voice

File: modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc [hunk 2]
```diff
@@ -34,9 +34,9 @@ that is not supported. The following TuneD plugins are currently not supported:
 * systemd
 
 
-[WARNING]
+[NOTE]
 ====
-The TuneD bootloader plugin is currently supported on {op-system-first} 8.x worker nodes. For {op-system-base-full} 7.x worker nodes, the TuneD bootloader plugin is currently not supported.
+The TuneD bootloader plugin is only supported on {op-system-first} worker nodes.
```
Comment (tmalove) [inline modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc:39]:
```suggestion
The TuneD boot loader plugin only supports {op-system-first} worker nodes.
```

**Review by tmalove — APPROVED** [review]

**Comment by sheriff-rh** [comment]
Merging!

File: modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc (hunk 2 above)
Comment (anildhingra) [reply modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc]: will it be supported on RHEL 8.x worker nodes ? or is it only for COREOS

**Comment by anildhingra** [comment]
will it be supported on RHEL 8.x worker nodes ? or is it only for COREOS

**Comment by sheriff-rh** [comment]
@anildhingra CoreOS only.

**Pull request merged**